tests/main/uc20-snap-recovery-encrypt/task.yaml

```diff
@@ -0,0 +1,120 @@
+summary: Integration tests for the snap-bootstrap binary
+
+# one system is enough, its a very specialized test for now
+systems: [ubuntu-19.10-64]
+
+debug: |
+    cat /proc/partitions
+
+restore: |
+    if [ -f loop.txt ]; then
+        losetup -d "$(cat loop.txt)"
+    fi
+
+prepare: |
+    echo "Create a fake block device image"
+    truncate --size=2GB fake.img
+
+    echo "Setup the image as a block device"
+    # without -P this test will not work, then /dev/loop1p? will be missing
+    losetup -fP fake.img
+    losetup -a |grep fake.img|cut -f1 -d: > loop.txt
+    LOOP="$(cat loop.txt)"
+
+    echo "Create an empty partition header"
+    echo "label: gpt" | sfdisk "$LOOP"
+
+    # XXX: update to the UC20 gadget when it's available
+    echo "Get the UC16 gadget"
+    snap download pc
+    unsquashfs -d gadget-dir pc_*.snap
+
+execute: |
+    LOOP="$(cat loop.txt)"
+    echo "Run the snap-bootstrap tool"
+    /usr/lib/snapd/snap-bootstrap create-partitions ./gadget-dir "$LOOP"
```

Comment:
One more remark (I added this earlier but it seems somehow GH or my browser ate it):
I wonder if we should skip all the generic create-partitions test because we already do this in uc20-snap-recovery and just test the bits in 71+ ?

Reply:
I think we should do that to avoid redundancy, but maybe we can do it when the other tests are updated to use the uc20 gadget? Doing this update is in my follow-up PR list.